Sdk/Track/Swrve.h

```diff
@@ -695,7 +695,7 @@ typedef void (^SwrveResourcesUpdatedListener) ();
 
 /*! Used internally to detect if the app is in the background.
  */
--(BOOL) appInBackground;
+-(BOOL) appInBackground __deprecated;
```

Comment:
It is an internal API so I think we can remove it without deprecating it first :boom: